### PR TITLE
Add sidebar background color on hover

### LINKED ISSUE #95: Add back in visibility of the version number to the docs sidebar
### Terms

- [X] I have searched all [open bug reports](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

One issues from the most recent changes to the documentation layout to add in the logo in #90 was that the version number is now not visible. As seen in the image below, there's a space between the logo and search bar where the version is:

Screenshot_2024-03-09 17 24 18

We could either change the text to black, or we can remove it entirely, but the former would be preferable :)

### PR DESCRIPTION
**Pull request by mhmohona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description
Based on [this comment](https://github.com/scribe-org/Scribe-Data/pull/92#issuecomment-1987057997) submitted a fix. 

### Related issue

Continuation of  #92 and fixes #95

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -180,7 +180,7 @@
 html_logo = "_static/ScribeDataLogo.png"
 html_theme_options = {
     "logo_only": True,
-    "display_version": False,
+    "display_version": True,
```
Comment (mhmohona) [inline docs/source/conf.py:183]: @andrewtavis here is the fix for #95. Apology for my mistake. :(

File: docs/source/_static/custom.css [hunk 2]
```diff
@@ -19,7 +19,11 @@
 .wy-nav-side .wy-menu-vertical a:hover {
   color: #fff !important;
 }
-
+/* Sidebar item background color on hover */
+.wy-nav-side .wy-menu-vertical li.toctree-l1.current>a:hover,
+.wy-nav-side .wy-menu-vertical li.toctree-l1>a:hover {
+    background-color: #333333 !important; /* Dark gray background */
```
Comment (wkyoshida) [inline docs/source/_static/custom.css:25]: nit - really quick, just noticed that the indent spacing here seems to be larger than in the others in the file

**Review by wkyoshida — APPROVED** [review]
Sweet!! :rocket: 

I'll let @andrewtavis take a look as well though

File: docs/source/conf.py (hunk 1 above)
Comment (andrewtavis) [reply docs/source/conf.py:183]: All good, @mhmohona! Happens :)

**Review by andrewtavis — APPROVED** [review]
Docs are really looking great, @mhmohona! Thanks so much for the hard work to get them up to such a standard :) :)

File: docs/source/_static/custom.css (hunk 2 above)
Comment (andrewtavis) [reply docs/source/_static/custom.css]: Done ✅

**Pull request merged**